internal/jobs/launch_instance_aws.go

```diff
@@ -49,7 +46,8 @@ func HandleLaunchInstanceAWS(ctx context.Context, job *worker.Job) {
 		return
 	}
 
-	ctx = contextLogger(ctx, job, args.AccountID, args.ReservationID)
+	logger := ctxval.Logger(ctx).With().Int64("reservation_id", args.ReservationID).Logger()
+	ctx = ctxval.WithLogger(ctx, &logger)
```

Comment:
This could have been written with `ptr.To` but this is okay too we use this all the time.